mempool/cat/pool.go

```diff
@@ -343,7 +343,7 @@ func (txmp *TxPool) TryAddNewTx(tx types.Tx, key types.TxKey, txInfo mempool.TxI
 	// Perform the post check
 	err = txmp.postCheck(wtx.tx, rsp)
 	if err != nil {
-		txmp.rejectedTxCache.Push(key, 0)
+		txmp.rejectedTxCache.Push(key, 0, "")
```

Comment:
why not have the `err` as part of the log?

Reply:
good point